workspaces/copilot/plugins/copilot-backend/README.md

```diff
@@ -15,10 +15,15 @@ To configure the plugin using the new backend system:
 
    const backend = createBackend();
 
-   backend.add(import('@backstage-community/plugin-copilot'));
+   backend.add(import('@backstage-community/plugin-copilot-backend'));
```

Comment:
Hey, when you want that this (good) change is visible on https://www.npmjs.com/package/@backstage-community/plugin-copilot-backend as well, then you should also add a changeset for this small change. :)

Reply:
Hi, done  - thanks :)